src/BizHawk.Emulation.Cores/Waterbox/WaterboxCore.cs

```diff
@@ -122,7 +122,7 @@ private void AdvanceRtc()
 		private WaterboxMemoryDomain[] _saveramAreas;
 		private int _saveramSize;
 
-		public unsafe bool SaveRamModified
+		public virtual unsafe bool SaveRamModified

```

Comment:
I did this to accommodate your overrides (`DOSBox.ISaveRam.cs`), but I'm not sure whether you should be doing that. Obviously all the other Waterbox cores make do with the base implementations.

Reply:
Exposing the save ram as memory domain instead would most likely be the correct way to handle saveram.

Reply:
Don't need to change this class anymore.